# Analyse de jeu de données : rendement agricole de la pomme de terre dans 6 pays européens (source : FAOSTAT)

In [4]:
import pandas as pd
data = pd.read_csv("FAOSTAT_data_fr_1-26-2025.csv")

Le jeu de données provient de **FAOSTAT**.

FAOSTAT fournit un accès gratuit aux statistiques sur l'alimentation et l'agriculture (y compris les sous-secteurs de la culture, de l'élevage, et de la foresterie) de plus de 245 pays et territoires et couvre tous les groupes régionaux de l'Organisation des Nations unies pour l'alimentation et l'agriculture depuis 1961 jusqu'à l'année disponible la plus récente.

## Analyse du jeu de données

In [5]:
data.head()

Code Domaine                       Domaine  Code zone  \
0          QCL  Cultures et produits animaux        106   
1          QCL  Cultures et produits animaux        150   
2          QCL  Cultures et produits animaux        203   
3          QCL  Cultures et produits animaux        229   
4          QCL  Cultures et produits animaux         68   

                                                Zone  Code Élément    Élément  \
0                                             Italie          5412  Rendement   
1                             Pays-Bas (Royaume des)          5412  Rendement   
2                                            Espagne          5412  Rendement   
3  Royaume-Uni de Grande-Bretagne et d'Irlande du...          5412  Rendement   
4                                             France          5412  Rendement   

   Code Produit          Produit  Code année  Année  Unité   Valeur Symbole  \
0           116  Pommes de terre        1961   1961  kg/ha  10383.2       A   
1           116  Pommes de terre        1961   1961  kg/ha  28039.6       A   
2           116  Pommes de terre        1961   1961  kg/ha  11812.6       A   
3           116  Pommes de terre        1961   1961  kg/ha  22478.5       A   
4           116  Pommes de terre        1961   1961  kg/ha  16099.2       A   

  Description du Symbole  Note  
0       Chiffre officiel   NaN  
1       Chiffre officiel   NaN  
2       Chiffre officiel   NaN  
3       Chiffre officiel   NaN  
4       Chiffre officiel   NaN

Le jeu de données contient des informations sur les rendements agricoles en kilogrammes par hectare (kg/ha) pour les pommes de terre (produit identifié par le code 116) dans différents pays européens de 1961 à 2023.

Chaque ligne correspond à une combinaison unique de pays, année et produit.

In [12]:
info = data.info()
print(info)

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 402 entries, 0 to 401
Data columns (total 15 columns):
 #   Column                  Non-Null Count  Dtype  
---  ------                  --------------  -----  
 0   Code Domaine            402 non-null    object 
 1   Domaine                 402 non-null    object 
 2   Code zone               402 non-null    int64  
 3   Zone                    402 non-null    object 
 4   Code Élément            402 non-null    int64  
 5   Élément                 402 non-null    object 
 6   Code Produit            402 non-null    int64  
 7   Produit                 402 non-null    object 
 8   Code année              402 non-null    int64  
 9   Année                   402 non-null    int64  
 10  Unité                   402 non-null    object 
 11  Valeur                  402 non-null    float64
 12  Symbole                 402 non-null    object 
 13  Description du Symbole  402 non-null    object 
 14  Note                    0 non-null      fl

Colonnes du jeu de données :

- Code Domaine et Domaine : Identifie le domaine d'étude, ici les cultures et produits animaux (code QCL).
- Code Zone et Zone : Identifie la zone géographique ou le pays concerné (ex. : France, Italie, Espagne, etc.).
- Code Élément et Élément : Décrit l'élément mesuré, ici le rendement (code 5412).
- Code Produit et Produit : Décrit le produit agricole concerné, ici les pommes de terre (code 116).
- Code Année et Année : Année de la mesure.
- Unité : Unité de la mesure, ici kg/ha.
- Valeur : Rendement mesuré en kg/ha.
- Symbole : Indique une source ou une spécificité de la valeur, ici A pour "chiffre officiel".
- Description du Symbole : Description associée au symbole.
- Note : Remarques ou commentaires additionnels (ici vide).

In [10]:
missing_values = data.isnull().sum()
print(missing_values)

Code Domaine                0
Domaine                     0
Code zone                   0
Zone                        0
Code Élément                0
Élément                     0
Code Produit                0
Produit                     0
Code année                  0
Année                       0
Unité                       0
Valeur                      0
Symbole                     0
Description du Symbole      0
Note                      402
dtype: int64


Le jeu de données présenté contient des informations sur les rendements agricoles en kilogrammes par hectare (kg/ha) pour les pommes de terre (produit identifié par le code 116) dans différents pays européens sur les années 1961 à 1963. Chaque ligne correspond à une combinaison unique de pays, année et produit.
Colonnes du jeu de données

    Code Domaine et Domaine : Identifie le domaine d'étude, ici les cultures et produits animaux (code QCL).
    Code Zone et Zone : Identifie la zone géographique ou le pays concerné (ex. : France, Italie, Espagne, etc.).
    Code Élément et Élément : Décrit l'élément mesuré, ici le rendement (code 5412).
    Code Produit et Produit : Décrit le produit agricole concerné, ici les pommes de terre (code 116).
    Code Année et Année : Année de la mesure.
    Unité : Unité de la mesure, ici kg/ha.
    Valeur : Rendement mesuré en kg/ha.
    Symbole : Indique une source ou une spécificité de la valeur, ici A pour "chiffre officiel".
    Description du Symbole : Description associée au symbole.
    Note : Remarques ou commentaires additionnels (ici vide).

Aperçu des données

    Période : Les données couvrent les années 1961 à 1963.
    Pays inclus : Italie, Pays-Bas, Espagne, Royaume-Uni, France, Allemagne.
    Produit : Les pommes de terre.
    Valeurs observées : Les rendements varient entre environ 9 453 kg/ha (Italie, 1962) et 30 497 kg/ha (Pays-Bas, 1962), avec des fluctuations annuelles.

Observations

    Les Pays-Bas montrent systématiquement les rendements les plus élevés pour les pommes de terre parmi les pays étudiés.
    L'Italie présente des rendements généralement inférieurs à ceux des autres pays européens mentionnés.
    Les données sont qualifiées comme étant des "chiffres officiels", ce qui assure une certaine fiabilité.

Utilisation possible

Ce jeu de données pourrait être utilisé pour :

    Analyser l'évolution des rendements agricoles des pommes de terre dans ces pays au début des années 1960.
    Comparer les pratiques agricoles ou les conditions climatiques entre les pays étudiés.
    Servir de base pour des études sur les facteurs influençant le rendement agricole à cette période.